E1a-ARSE/ALIN01/Dataset/VerticalAlignmentParameters.csv

```diff
@@ -1,18 +1,18 @@
-440.109;64.029;41.464;0.0486302;-0.0154579;-1000.0;.CIRCULARARC.
-504.138;203.572;42.525;-0.0154579;-0.0154579;;.CONSTANTGRADIENT.
-707.709;55.848;39.378;-0.0154579;-0.0714863;-1000.0;.CIRCULARARC.
-763.558;76.625;36.952;-0.0714863;0.0053208;1000.0;.CIRCULARARC.
-840.183;152.345;34.421;0.0053208;0.0053208;;.CONSTANTGRADIENT.
-992.528;36.817;35.232;0.0053208;0.0237361;2000.0;.CIRCULARARC.
-1029.345;52.563;35.766;0.0237361;-0.0288451;-1000.0;.CIRCULARARC.
-1081.907;188.502;35.632;-0.0288451;-0.0288451;;.CONSTANTGRADIENT.
+Name;Type;StartDistAlong;HorizontalLength;StartHeight;StartGradient;EndGradient;Radius;Type
+V1;IfcAlignmentVerticalSegment;0;67,9666275938242;43,475329;-0,00773410503537014;-0,00773410503537014;0;CONSTANTGRADIENT
+V2;IfcAlignmentVerticalSegment;67,9666275938242;25,56006049186;42,9496679632888;-0,00773410503537014;-0,00773410503537014;-300;CIRCULARARC
+V3;IfcAlignmentVerticalSegment;93,5266880856841;20,3985317664884;41,6603192684821;-0,0933380176556316;-0,0933380176556316;0;CONSTANTGRADIENT
+V4;IfcAlignmentVerticalSegment;113,925219852173;21,6842049028376;39,7563607503138;-0,0933380176556316;-0,0933380176556316;300;CIRCULARARC
+V5;IfcAlignmentVerticalSegment;135,60942475501;68,6257430776865;38,5220312181138;-0,0206577987550277;-0,0206577987550277;0;CONSTANTGRADIENT
+V6;IfcAlignmentVerticalSegment;204,235167832697;56,9319301335483;37,1043744281956;-0,0206577987550277;-0,0206577987550277;2000;CIRCULARARC
+V7;IfcAlignmentVerticalSegment;261,167097966245;46,449675607139;36,7388035371268;0,00781281114018211;0,00781281114018211;0;CONSTANTGRADIENT
+V8;IfcAlignmentVerticalSegment;307,616773573384;101,90056287711;37,101706080191;0,00781281114018211;0,00781281114018211;2500;CIRCULARARC
+V9;IfcAlignmentVerticalSegment;409,517336450494;30,5916549869268;39,9762969622553;0,0486301987513986;0,0486301987513986;0;CONSTANTGRADIENT
+V10;IfcAlignmentVerticalSegment;440,108991437421;64,0288214133387;41,4639752243932;0,0486301987513986;0,0486301987513986;-1000;CIRCULARARC
+V11;IfcAlignmentVerticalSegment;504,13781285076;203,571683667725;42,5248787225931;-0,0154578700237117;-0,0154578700237117;0;CONSTANTGRADIENT
+V12;IfcAlignmentVerticalSegment;707,709496518485;55,8482922569574;39,3780940959472;-0,0154578700237117;-0,0154578700237117;-1000;CIRCULARARC
+V13;IfcAlignmentVerticalSegment;763,557781493212;76,6250733893689;36,9521538451958;-0,0714862764285531;-0,0714862764285531;1000;CIRCULARARC
+V14;IfcAlignmentVerticalSegment;840,182854882581;152,344654262115;34,4209169381877;0,00532083289209283;0,00532083289209283;0;CONSTANTGRADIENT
+V15;IfcAlignmentVerticalSegment;992,527509144696;36,8173619984802;35,2315173854975;0,00532083289209283;0,00532083289209283;2000;CIRCULARARC
+V16;IfcAlignmentVerticalSegment;1029,34487722718;52,5625639402238;35,7663724122683;0,0237361222562795;0,0237361222562795;-1000;CIRCULARARC
+V17;IfcAlignmentVerticalSegment;1081,90744116741;188,501540832592;35,6321941963812;-0,0288451180418207;-0,0288451180418207;0;CONSTANTGRADIENT
```

Comment:
Same problem here as well - Start/End-Gradient for CircularArcs are equal?

Reply:
I think there must be a problem in my code extracting these parameters. Will check. Thanks "hawkeye"...

Reply:
Can you check now?

Reply:
LFTM